README.md

```diff
@@ -0,0 +1,63 @@
+## Contract deployment
+
+Before compiling and deploying your contract you need to create a `.env` file. 
+See `.env.example` for this purpose.
+
+Then, you can start the docker containers to create your development environment.
+It includes a **ganache** instance and a **python+solidity** container to run the code, compile contracts etc.
+
+```bash
+docker compose up --build
+```
+
+If you don't use the docker containers, remember to change your environment variables, and the **ENV_PATH** variable in `python/deployment/utils.py`.
```

Comment:
I do not think forcing user to change path in python file is good idea. consider splitting .env file into 2 files - one in the root folder and second one in python/deployment it is independent from the environment that it is running